AY2021S1-CS2103-T16-3/tp
# Functionality Increment: Statistics piechart for room allocations

### What this does
This PR adds the functionality increment for the statistics piechart for room allocations. The piechart shows the ratio of allocated and unallocated rooms in ResiReg.


Part of #13.

### How to test

1. Launch the application.
2. Click the Statistics tab.
3. You should see a pie chart representing the number of allocated and unallocated rooms.
3. Run `allocate si/3 ri/3`.
4. Notice an update to the pie chart that now has 1 more allocated and 1 less unallocated room.

### Notes
![image](https://user-images.githubusercontent.com/27071473/97330838-afa57180-18b3-11eb-9a61-605e184c4859.png)
